apps/yapms/src/lib/utils/patterns.ts

```diff
@@ -35,3 +35,21 @@ export function makePattern(candidates: Array<RegionCandidate>) {
 	mapSVG?.appendChild(pattern);
 	return `url(#${name})`;
 }
+
```

Comment:
This is the `makePatternCSS` function as noted above. This generates the same striped pattern used for split regions, but in CSS `linear-gradient` form so it can be used on HTML elements.